src/visidentifier/visconfig.hpp

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (C) 2024 Renesas Electronics Corporation.
+ * Copyright (C) 2024 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef VISCONFIG_HPP_
+#define VISCONFIG_HPP_
+
+#include <string>
+
+#include <Poco/Dynamic/Var.h>
+
+#include <aos/common/tools/error.hpp>
+
+#include "utils/time.hpp"
+
+/***********************************************************************************************************************
+ * Static
+ **********************************************************************************************************************/
+
+static constexpr const char* cVisServerTagName        = "visServer";
+static constexpr const char* cCaCertFileTagName       = "caCertFile";
+static constexpr const char* cWebSocketTimeoutTagName = "webSocketTimeout";
```

Comment:
i think this static will be better to put in private section class VISConfig

Reply:
The idea was to use this constants for unit test as well (instead of magic strings). 
But I can refactor if you insist